packages/react-components/react-table/src/stories/Table/SizeSmall.stories.tsx

```diff
@@ -77,11 +77,7 @@ export const SizeSmall = () => {
             <TableCell>
               <TableCellLayout
                 media={
-                  <Avatar
-                    name={item.author.label}
-                    badge={{ status: item.author.status as PresenceBadgeStatus }}
-                    size={24}
-                  />
+                  <Avatar name={item.author.label} badge={{ status: item.author.status as PresenceBadgeStatus }} />
```

Comment:
(nit) This is unrelated to this change, but we should encourage `as` casts like this in stories. You could either define the entire items array `as const`, or possibly adding making the individual statuses as const in the object, like `status: 'available' as const`

Reply:
Fixed in https://github.com/microsoft/fluentui/pull/24807/commits/bcf7dab00a29b65c3c0b59bf6a9312f8d4414700

Reply:
Thanks! I just realized I left out "not" in "should _not_ encourage" 🤦‍♂️ but I think you figured it out 🙂